WordPress/Sniff.php

```diff
@@ -3385,4 +3385,38 @@ protected function get_list_variables( $stackPtr, $list_open_close = array() ) {
 		return $var_pointers;
 	}
 
+	/**
+	 * Check whether a function has been marked as deprecated via a @deprecated tag
+	 * in the function docblock.
+	 *
+	 * {@internal This method is static to allow the ValidFunctionName class to use it.}}
```

Comment:
```suggestion
	 * {@internal This method is static to allow the ValidFunctionName class to use it.}
```

Reply:
https://docs.phpdoc.org/references/phpdoc/tags/internal.html

> The inline version of this tag may, contrary to other inline tags, contain text but also other inline tags. To increase readability and ease parsing should the tag be terminated with a double closing brace, instead of a single one.

Reply:
Oh wasn't aware of this. Thanks for the link with the explanation :+1: